src/js/components/FormGroupComponent.jsx

```diff
@@ -53,7 +53,7 @@ var FormGroupComponent = React.createClass({
     var child = React.Children.only(props.children);
     var formControlChild = React.cloneElement(
       child, {
-        className: "form-control",
+        className: "form-control mousetrap",
```

Comment:
No problem by explaining what it does I realised that this is true for all key bindings and I prevented a very bad bug where you always have to refocus a input field after pressing s :-)